src/context/CaptureDetailContext.js

```diff
@@ -36,15 +41,45 @@ export function CaptureDetailProvider(props) {
 
   // EVENT HANDLERS
 
-  const getCaptureDetail = async (id) => {
-    if (id == null) {
-      log.debug('getCapture called with no id');
-      return Promise.resolve(STATE_EMPTY.capture);
-    } else {
-      return api.getCaptureById(id).then((capture) => {
-        setState({ ...state, capture });
-        return capture;
-      });
+  const getCaptureDetail = async (capture) => {
+    log.debug('getCaptureDetail:', {
+      reference_id: capture.reference_id,
+      id: capture.id,
+    });
+
+    // getOrganization gets the wrong org when this is in the global context
+    const BASE_URL = {
+      LEGACY: `${API_ROOT}/api/${getOrganization()}trees/`,
+      CAPTURE_MATCH: `${TREETRACKER_API}/captures?reference_id=`,
+    };
+
+    try {
+      if (!capture?.id) {
+        log.debug('getCapture called with no reference_id');
+      } else {
+        // NOTE: The reference_id in the new api === the id in the old api
+        const query = `${
+          BASE_URL[capture.reference_id ? 'CAPTURE_MATCH' : 'LEGACY']
+        }${capture.reference_id || capture.id}`;
+
+        fetch(query, {
+          headers: {
+            Authorization: session.token,
+          },
+        })
+          .then(handleResponse)
+          .then((data) => {
+            if (data.captures) {
+              // new treetracker-api
+              setState({ ...state, capture: data.captures[0] });
+            } else {
+              // legacy api
+              setState({ ...state, data });
+            }
```

Comment:
The legacy api doesn't return the capture nested in an object so we need this to handle the difference.